ipc_plugin/ipc_mod.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python3.5m
+#
+#   Copyright (C) 2017, Collabora Ltd.
+#
+#   This Source Code Form is subject to the terms of the Mozilla Public
+#   License, v. 2.0. If a copy of the MPL was not distributed with this file,
+#   You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+
+import zmq
+
+IPC_SOCKET = "ipc:///tmp/vsi.ipc"
```

Comment:
The name 'vsi' is not good. Also, using a hardcoded name in /tmp seems like bad practice. Let's either use a well-known name in /run or a tmp file with a generated name?